src/kernel/src/main.rs

```diff
@@ -91,14 +91,14 @@ fn main() -> ExitCode {
     info!("Mounting / to {}.", args.system.display());
 
     if let Err(e) = fs.mount(VPathBuf::new(), args.system) {
-        error!(e, "Mount failed");
+        error!(e, "Mount failed!");
```

Comment:
All error messages will be appended with `: reason` that means this will output `Mount failed!: reason` instead of `Mount failed: reason`, which looks weird.